.github/workflows/deploy-to-development.yml

```diff
@@ -0,0 +1,77 @@
+name: Deploy to Development
+
+on:
+  push:
+    branches-ignore:
+      - main
+  pull_request:
```

Comment:
Do we need pull_request and push event handling?

Reply:
Oh no I don't - I was basically thinking that we probably don't want the pull_request event trigger and can just leave the push one in